### PR TITLE
Add service map

### PR DESCRIPTION
**Pull request by frankieroberto**
Adds the current service map as a reference, so that you can see how BAT services relate to other teacher services. Also lets people download the PDF.

Probably this could ultimately 'live' somewhere else, as it's broader than just BAT, but in the meantime at least this would let people access it.

I've added an HTML version of the diagram underneath it, for accessibility and so that we can link to the actual services.

➡️ [Preview](https://deploy-preview-564--bat-design-history.netlify.app/service-map/)

### CODE REVIEW

File: app/service-map.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+---
+layout: page
+title: Service map
```
Comment (ralph-hawkins) [inline app/service-map.md]: I'd argue this wasn't a 'service map' as most people would expect. Maybe call it the 'Teacher services diagram'?

File: app/service-map.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+description: How the ‘becoming a teacher’ services relate to other services in the teaching journey
+tags: reference
+eleventyComputed:
+  eleventyNavigation:
+    key: "{{ title }}"
+    excerpt: "{{ description }}"
+    parent: home
+---
+
+The 'becoming a teacher' services are part of a broader range of digital services which encompass the whole teaching journey, from initial interest through to retirement.
```
Comment (ralph-hawkins) [inline app/service-map.md:13]:
```suggestion
The ‘becoming a teacher’ services are part of a broader range of digital services which encompass the whole teaching journey, from initial interest through to retirement.
```

**Review by ralph-hawkins — CHANGES_REQUESTED** [review]
I find the HTML version pretty hard to read — could you use a table or CSS grid to add horizontal alignment?

File: app/service-map.md (hunk 1 above)
Comment (frankieroberto) [reply app/service-map.md]: Map and diagram feel fairly synonymous to me? (eg "tube map" / "mind map"). It's probably technically a diagram, but I think most people call it a map? 

How about "Teacher services map"?

**Comment by frankieroberto** [comment]
> I find the HTML version pretty hard to read — could you use a table or CSS grid to add horizontal alignment?

I started with a table, but then felt that it'd probably need another column of headers to the left, which was getting a bit tight. Also worried that it was abusing tables for layout a bit...

I'll have a look at CSS grids though.

File: app/service-map.md (hunk 1 above)
Comment (ralph-hawkins) [reply app/service-map.md]: Yeah, that works

**Comment by frankieroberto** [comment]
> > I find the HTML version pretty hard to read — could you use a table or CSS grid to add horizontal alignment?
> 
> I started with a table, but then felt that it'd probably need another column of headers to the left, which was getting a bit tight. Also worried that it was abusing tables for layout a bit...
> 
> I'll have a look at CSS grids though.

@ralph-hawkins I've done the simplest thing I could think of and have added bullets to the lists, which hopefully separates them from the lead-in lines a bit more:

Screenshot 2022-02-28 at 16 29 24

**Review by ralph-hawkins — APPROVED** [review]

**Pull request merged**